### PR TITLE
Handle no changepoint case in do_prophet

### PR DESCRIPTION
**Pull request by hideaki**
### Description
Handle no changepoint case in do_prophet

### Checklist

Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [ ] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [x] Pass devtools::test()
- [ ] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

File: R/prophet.R [hunk 1]
```diff
@@ -204,12 +204,18 @@ do_prophet_ <- function(df, time_col, value_col = NULL, periods, time_unit = "da
...
     }
     # TODO: Maybe we should take average when MCMC is used and there are multiple delta values for each channge point.
-    changepoints_df <- data.frame(ds = m$changepoints, slope_change = m$params$delta[1,])
-    # m$changepoints is POSIXct. Cast it to Date when necessary so that left_join works.
-    if (lubridate::is.Date(aggregated_data$ds)) {
-      changepoints_df$ds <- as.Date(changepoints_df$ds)
+    if (!is.numeric(m$changepoints)) { # m$changepoints seems to become numeric single 0 when empty.
+      changepoints_df <- data.frame(ds = m$changepoints, slope_change = m$params$delta[1,])
+      # m$changepoints is POSIXct. Cast it to Date when necessary so that left_join works.
+      if (lubridate::is.Date(aggregated_data$ds)) {
```
Comment (kei51e) [inline R/prophet.R]: Shouldn't be `is.POSIXct`?

Comment (hideaki) [reply R/prophet.R]: It is is.numeric. I tried to explain why the comment.

Comment (kei51e) [reply R/prophet.R]: ? I'm talking about this part. 

image

Comment (hideaki) [reply R/prophet.R]: I see. Updated the comment so that this is less confusing.

**Pull request merged**